src/app/index.js

```diff
@@ -1,69 +1,208 @@
-var slideIndex, slides, dots;
-window.onload = function() {
+/**============= Require all the scss and js files =================== */
+import '../../style.scss';
+import { makeRequest } from '../app/core/services/sabkabazar.service';
+import { home } from '../app/home/home';
+import { products, onClickCategory } from '../app/products/products';
+import { login } from '../app/login/login';
+import { signup } from '../app/signup/signup';
 
-  slides = document.getElementsByClassName("mySlides");
-  dots = document.getElementsByClassName("dot");
+const WINDOW_CONFIG = window['SABKABAZAR_CONFIG'];
 
-  content.innerHTML = routes['home'];
-  this.slideIndex = 1;
-  this.showSlides(this.slideIndex);
+let slideIndex, slides, dots, banners, categories, productList;
+
+/**============= Handling page change on navigation click keeping header/footer fixed =================== */
+const routes = {
+  'signup': signup,
+  'signin': login,
+  'home': home,
+  'products': products
+}
+
+$(document).ready(function() {
+
+  let handlebarsData;
+
+  // fetch banners and then categories
+  makeRequest(`${WINDOW_CONFIG.apiUrl}/mock-data/banners/banners.json`, 'GET')
```

Comment:
This code is not organized correctly. There should have an independent module handling this bootstrapping code; here you could have just invoked it.